src/book/02-system/04-hardware/01-overview.mdx

```diff
@@ -18,16 +18,20 @@ The NUbots team uses the NUgus hardware platform, a modification of the [igus®
 
 The following modifications were made to fit our specific requirements:
 
-- The components of the robot were split to to fit the size of the printer bed and volume.
-- The torso was modified to have open shoulder covers for shoulder servo cable access.
+- Split components of the robot to fit the size of the printer bed and volume.
+- Modified the torso to have open shoulder covers for shoulder servo cable access.
+- Redesigned the legs with a simpler model made out of aluminium, with XH540-W270-R servos rather than MX-106 servos, and onyx connectors for the hip and ankle blocks and onyx supports inside the legs.
+- Moved the buttons to the panel above the removable back panel to prevent issues with button cables when removing the back panel.
+- Added an aluminium bar between the hip yaw servos to reduce flex.
+- Created custom draws for the computer (intel NUC) and subcontrollers (OpenCR, NUSense)
 
```

Comment:
- Designed a bearing housing for the hip yaw that takes a tapered roller bearing from a mountain bike head stock. This interfaces between the servo shaft and the hip pitch/roll block and takes the load of the servo shaft and spreads it over a wider area on the bottom of the torso.